lib/inferno/entities/test_suite.rb

```diff
@@ -108,6 +108,11 @@ def configuration_messages(new_messages = nil, force_recheck: false)
             end
         end
 
+        # @private
+        def suite_requirements(suite_options)
+          children(suite_options).flat_map(&:verifies_requirements)
```

Comment:
`children` is only direct children, not all descendents, so we'll need a recursive method to traverse them all.